UPGRADING

```diff
@@ -79,6 +79,12 @@ Changes from 2.8.0 to 2.8.1
   resolves to several addresses (and would only listen on the first hit,
   as it did before in such cases) [#2012]
 
+- A definitive behavior for `LISTEN *` directives became specified, to try
+  handling both IPv4 and IPv6 "any" address (subject to `upsd` CLI options
+  to only choose one, and to OS abilities); this use-case may be practically
+  implemented as a single IPv6 socket on systems with enabled "dual-stack"
```

Comment:
same comment

It strikes me as odd to have repeated content in UPGRADING as is in NEWS, but whatever.

Reply:
My take is that NEWS cover the changelog feature-wise, and UPGRADING highlights potentially breaking changes (e.g. "if you depended on this old behavior, check if you need to reconfigure now").

Reply:
If you are editing all this, your call.    Just pointing out my opinion. (I personally would leave things out of UPGRADING that are not expected to cause problems with normal usage, as part of weighing maintenance burden more heavily than theoretical user trouble.)